### DISCUSSION TITLE
应用实践

### DISCUSSION THREAD

**Discussion by wlkfec** [kubeedge/kubeedge#4203] [Q&A]
感觉整体设计与k8s apiz资源紧密联系，那么在实践中如何解决后端应用与api之间的联系，比如说是否需要频繁操作device api去更新设备状态

**Comment by vincentgoat** [comment]
The control surface is decoupled from the data surface, so there will not be frequent operations on `device api`.
cc @RyanZhaoXB

**Reply by wlkfec** [reply]
我的问题有点歧义，我查看文档以及代码中的示例，发现设备创建以及设备状态更新都是直接操作k8s中的api资源抽象的，那么这样的话，这种模式不是就和传统的后端服务不符，因为其把相关的资源对象都容纳进k8s内部了，传统的话更多的是依赖各种数据库以及自定义业务逻辑
